e2e/app/page/data-page.ts

```diff
@@ -21,8 +24,11 @@ export default class DataPage extends AuthenticatedPage {
 
   async isLoaded(): Promise<boolean> {
     try {
-      await this.waitUntilTitleMatch('Data Page');
-      await this.page.waitForXPath(TAB_SELECTOR.showAllTab, {visible: true});
+      await Promise.all([
+        this.waitUntilTitleMatch(PAGE.TITLE),
+        this.page.waitForXPath(TAB_SELECTOR.showAllTab, {visible: true}),
+        this.waitUntilNoSpinner(),
+      ]);
       return true;
```

Comment:
All `isLoaded` methods refactored to accommodate spinner on blank page. See screenshot in PR description.